docs/tutorials/structure_function_showcase.ipynb

```diff
@@ -10,33 +10,85 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    # Create ColumnMapper object that will tell ensamble
</pre><p>ensamble -&gt; ensemble</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tape/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>